Dockerfile

```diff
@@ -68,6 +68,7 @@ RUN apt-get update \
         python3-dev \
         zlib1g-dev \
     && pip install openupgradelib \
+    && pip install boto3 \
```

Comment:
Use requirements.txt instead

Reply:
i.e.

     && pip install --no-cache-dir -r https://raw.githubusercontent.com/it-projects-llc/misc-addons/${ODOO_BRANCH}/requirements.txt \